condor_watch_q.py

```diff
@@ -1019,9 +1019,9 @@ def make_progress_bar(totals, width=None, color=True):
         "".join(
             colorize(char * length, c) if color else char * length
             for char, length, c in zip(
-                ("!", "-", "=", "#"),
+                ("#", "=", "-", "!"),
                 bar_section_lengths,
-                (Color.RED, Color.BRIGHT_YELLOW, Color.CYAN, Color.GREEN),
+                (Color.GREEN, Color.CYAN, Color.BRIGHT_YELLOW, Color.RED),
```

Comment:
I don't like that we had to change multiple places for consistency. Let's wrap the characters and colors up in a dictionary of tuples somewhere, like
```python
PROGRESS_BAR_CHARS_AND_COLORS = {
    JobStatus.COMPLETED: (Color.GREEN, "#"),
    ...
}
Not quite sure how to handle the fact that one of the progress bar groups has multiple job statuses in it... maybe the keys should themselves be tuples of `JobStatus`? Play around; the key thing is to try to get the entire "definition of the progress bar" in one place. Keep in mind that dictionaries are not ordered in Python 2, so you can't rely on the key ordering you write into the file to specify the order of the parts of the progress bar.